### DISCUSSION TITLE
Prevent form submission on enter in the field layout designer's search field

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#11415] [Ideas] [answered]
This an extremely minor thing, but hitting the Enter key in a field layout's edit form, while focus is on the field library search input, will submit the primary form (i.e. save the element source/field layout). 

I think that probably shouldn't happen, as I suspect the user's intention with that particular Enter keystroke is _most likely_ not to actually save the element source/field layout; probably it's either going to be entirely accidental or meant to (needlessly) "submit" the field library search form itself.

**Comment by brandonkelly** [comment, marked as answer]
Good call! Just addressed this for the next v3 and v4 releases (28910a33d014945d157a5f9680267cad939bad36).

**Reply by brandonkelly** [reply]
Craft 3.7.45 and 4.0.5 are out now with that change.
